Installing-and-Configuring/OS/index.md

```diff
@@ -58,19 +58,27 @@ Your other partitions may be encrypted or not based on your preferences and this
+In this Disk Unlock Key scenario, that key is provided to the OS in the form of a transitional crypttab file, fed to the operating system inside of the initrd file prior of the Operating system booting. Consequently, the Operating system tries this key which match required decryption key for slot 1 and the OS continues to boot without prompting the user for the Disk Recovery Key passphrase.
+
+If the Heads TPM drive decryption feature is not disabled in board configuration (enabled by default if TPM enabled board), Heads will [prompt](/Keys/#tpm-disk-encryption-key) for a Disk Unlock Key passphrase (which might be a PIN: A short passphrase/password) when [saving](https://github.com/osresearch/heads/blob/c3b0bd6ffbe816430dd41ef54e649af52ed1ff3b/initrd/bin/kexec-save-default#L52) a new default boot option configuration. This passphrase is different then the Disk Recovery Key passphrase, which was entered at OS installation for the LUKS encrypted container. The TPM Disk Unlock passphrase will be used to seal/unseal TPM kept Disk Unlock Key, which unlocks LUKS key slot 1 for the root partition when the OS boots, after having [injected](https://github.com/osresearch/heads/blob/c3b0bd6ffbe816430dd41ef54e649af52ed1ff3b/initrd/bin/kexec-insert-key#L75-L80) a crafted initrd into OS boot process. The randomized 128 characters [LUKS Disk Unlock Key](https://github.com/osresearch/heads/blob/c3b0bd6ffbe816430dd41ef54e649af52ed1ff3b/initrd/bin/kexec-seal-key#L55-L60) generated from Heads linux random number generator to create the key in slot 1 and [sealed in the TPM NV memory space](https://github.com/osresearch/heads/blob/c3b0bd6ffbe816430dd41ef54e649af52ed1ff3b/initrd/bin/kexec-seal-key#L105-L118) and [unsealed](https://github.com/osresearch/heads/blob/c3b0bd6ffbe816430dd41ef54e649af52ed1ff3b/initrd/bin/kexec-unseal-key) only if previous measurements that happened at [sealing](https://github.com/osresearch/heads/blob/c3b0bd6ffbe816430dd41ef54e649af52ed1ff3b/initrd/bin/kexec-seal-key#L105-L117) are consistent (LUKS header, firmware measurement) for which Disk Unlock Key passphrase will [unseal](https://github.com/osresearch/heads/blob/c3b0bd6ffbe816430dd41ef54e649af52ed1ff3b/initrd/bin/kexec-unseal-key#L20-L43) TPM NV reserved memory kept secret in that case.
+
+This option adds an extra layer of security. The TPM will only release the LUKS Disk Unlock key if the measurements match (again, LUKS header, firmware measurements and Disk Unlock Key passphrase releasing TPM VN stored Disk Unlock key, to be injected as an additional initrd file, decompressed by OS on initial boot step and used to unlock the LUKS container prior proceeding, if and only if everything is congruent). Let's remember that booting a default boot option also verifies /boot detached signed digest prior of offering to unlock encrypted storage with a Disk Unlock Key passphrase.
```

Comment:
This option adds an extra layer of security. The TPM will only release the LUKS Disk Unlock key if the measurements match (again, LUKS header, firmware measurements and Disk Unlock Key passphrase releasing TPM NV stored Disk Unlock key, to be injected as an additional initrd file, decompressed by OS on initial boot step and used to unlock the LUKS container prior proceeding, if and only if everything is congruent). Let's remember that booting a default boot option also verifies /boot detached signed digest prior of offering to unlock encrypted storage with a Disk Unlock Key passphrase.****

Reply:
dont see a difference here except for the four **** at the end. I updated anyway.